snippets/idms-global-pull-secret.adoc

```diff
@@ -3,10 +3,25 @@
 // * modules/builds-image-source
 // * modules/images-configuration-registry-mirror
 
-:_mod-docs-content-type: SNIPPET
+ifeval::["{context}" == "enabling-windows-container-workloads"]
+:winc:
+endif::[]
 
+:_mod-docs-content-type: SNIPPET
 
+ifndef::winc[]
 [NOTE]
 ====
 If your cluster uses an `ImageDigestMirrorSet`, `ImageTagMirrorSet`, or `ImageContentSourcePolicy` object to configure repository mirroring, you can use only global pull secrets for mirrored registries. You cannot add a pull secret to a project.
 ====
+endif::winc[]
+ifdef::winc[]
+[NOTE]
+====
+If your cluster uses an `ImageDigestMirrorSet` or `ImageTagMirrorSet` object to configure repository mirroring, you can use only global pull secrets for mirrored registries. You cannot add a pull secret to a project.
```

Comment:
> You cannot add a pull secret to a project.

@mansikulkarni96 Is this accurate?

Reply:
I haven't tried doing this but this is a generic note and not specific to Windows 
It comes from existing docs https://docs.openshift.com/dedicated/openshift_images/image-configuration.html

Reply:
We don't need to specify this note under enabling workloads.

Reply:
>We don't need to specify this note under enabling workloads.

@mansikulkarni96 We don't need the _You cannot add a pull secret to a project._ sentence, or the whole [NOTE]?

Reply:
@mansikulkarni96 Can you help with ^^

Reply:
@mburke5678 sorry I missed this comment, I think we can remove the whole note.